project_setup/scripts/skeleton

```diff
@@ -65,12 +70,20 @@ def clone_repo(parent_repo, new_repo, org, dir=None):
         f"git remote add origin git@github.com:{org}/{new_repo}.git",
         "Add a new remote origin for the this repository.",
     )
-    run(f"git tag -d $(git tag -l)", f"Delete all local git tags from {parent_repo}")
+    run("git tag -d $(git tag -l)", f"Delete all local git tags from {parent_repo}")
     run(
         fr"find . \( ! -regex '.*/\.git/.*' \) -type f -exec "
         fr"perl -pi -e s/{parent_repo}/{new_repo}/g {{}} \;",
         "Search and replace repository name in source files.",
     )
+    lineage = {
+        "version": 1,
```

Comment:
Perhaps this version number should be a variable defined up by `LINEAGE_CONFIG` ?  Maybe, maybe not- your call.

Reply:
Good idea!  It is quite magical down there. 🧙